<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListManage</class>
 <widget class="QWidget" name="ListManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QTableView" name="list_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sqlspace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
  </widget>
  <widget class="QComboBox" name="listType">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>6</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>黑名单</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>白名单</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 11pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>黑白名单选择：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUserBtn">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>成员添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delUserBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>成员删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
